components/List/BUILD

```diff
@@ -58,3 +59,28 @@ mdc_objc_library(
     ],
     visibility = ["//visibility:public"],
 )
+
+mdc_objc_library(
+    name = "unit_test_sources",
+    testonly = 1,
+    srcs = native.glob([
+        "tests/unit/*.m",
+        "tests/unit/*.h",
```

Comment:
Tip: I believe this can be removed unless you plan to add an umbrella test file.